graphql_server/flask/blueprint.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+
+from flask import Blueprint
+
+from .graphqlview import GraphQLView
+
+
+class GraphQL(object):
+    def __init__(self, app, schema, **options):
+        self.app = app
+        warnings.warn(
+            "GraphQL Blueprint is now deprecated, please use GraphQLView directly"
+        )
```

Comment:
If this is deprecated in the current flask-graphql package then this seems like a good time to remove it completely.

Reply:
YEET! Consider it done!